Makefile

```diff
@@ -10,11 +10,11 @@ init:
 
 .PHONY: install
 install: init
-	stack install --pedantic --test --local-bin-path=dist
+	stack install --pedantic --test --no-run-tests --local-bin-path=dist
```

Comment:
The docs told me to do `make services` so that's what I did. Then it was running tests all the time, it was annoying, and I asked Tiago whether I could remove it. There wasn't much reasoning involved :slightly_smiling_face:

I don't have a strong opinion about running vs. not running them on `install`, but it seems counter-intuitive to me that `install` should test anything, so I'm +0.5 on keeping the change and updating the CI instead. If you're okay with it, I'll do it.

Reply:
In my opinion, there should be one way to compile faster (no optimizations, no tests), and one way to compile slower (with more validations, i.e. tests; and resulting in optimized binaries). That's currently provided via `make`==`make fast` and `make install` respectively. `make services` depends on `make install`. 

As I said, I think removing the tests on `make install` is dangerous, as it means they will probably never run (people will forget to run them - nothing indicates they can, or they should - the docs don't mention anything here). I'm not sure why you needed to run `make services` repeatedly, you were probably only trying to compile nginz (which a `cd nginz && make` would do, too, and be faster.)

Reply:
See #312.

Reply:
> As I said, I think removing the tests on make install is dangerous, as it means they will probably never run (people will forget to run them [...])

But if CI runs tests (as it should), why do people have to run them too?